src/components/UIButton/UIButton.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```

Comment:
I created a new Button component because the current one has a bunch of cruft in it that's not relevant. We should use this button for the rounded green and white Hylo style buttons in the designs.